src/QuantumOptics.jl

```diff
@@ -7,6 +7,7 @@ using SparseArrays, LinearAlgebra
 export
     ylm,
     eigenstates, eigenenergies, simdiag,
+    LapackDiag, ArpackDiag, KrylovDiag,
```

Comment:
Given that the library mostly automatically decides which method to use, I think it would be best to not export these. If nonetheless we want them exported, they should at least have a docstring each.